src/components/SelectComponents/Autosuggest/SuggestionTag/__tests__/SuggestionTag.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import { SuggestionTag } from '../SuggestionTag';
+
+describe('SuggestionTag', () => {
+    it('should render nothing if no children are provided', () => {
+        const view = render(<SuggestionTag />);
+
+        expect(view.container).toMatchSnapshot();
+    });
+
+    it('should render if children is 0', () => {
+        const view = render(<SuggestionTag>{0}</SuggestionTag>);
+
+        expect(view.container).toMatchSnapshot();
+        expect(screen.getByText('0')).toBeInTheDocument();
+    });
+
+    it('should render correctly', () => {
```

Comment:
what is the meaning of rendering correctly?

If we don't provide onClick it will never show a button which is what is being expected here but it will render correctly if we only pass a child. 

Maybe it is better to also expect the child value that is being passed.

Reply:
:+1: